blivet/devices/lvm.py

```diff
@@ -966,6 +968,20 @@ def dependsOn(self, dep):
         return (self.origin == dep or
                 super(LVMSnapShotBase, self).dependsOn(dep))
 
+    def readCurrentSize(self):
+        log_method_call(self, exists=self.exists, path=self.path,
+                        sysfsPath=self.sysfsPath)
+        size = Size(0)
+        if self.exists and os.path.exists(self.path) and \
+           os.path.isdir(self.sysfsPath):
+            cowSysfsPath = util.get_cow_sysfs_path(self.path, self.sysfsPath)
+
+            if os.path.exists(cowSysfsPath) and os.path.isdir(cowSysfsPath):
```

Comment:
You should be able to omit ```os.path.exists(cowSysfsPath)``` since ```os.path.isdir(cowSysfsPath)``` will evaluate to False if ```cowSysfsPath``` does not exist.